DB/src/main/java/io/deephaven/db/tables/lang/DBLanguageParser.java

```diff
@@ -1528,6 +1531,11 @@ public Class visit(MethodCallExpr n, VisitArgs printer) {
 
         //now do some parameter conversions...
 
+        Class methodClass = variables.get(n.getName());
+        if (methodClass == NumbaCallableWrapper.class) {
+            parsePyNumbaVectorizedFunc(n, expressions, expressionTypes);
```

Comment:
We aren't really "parse"ing the func, are we? There aren't any side-effects, nor return value, correct? Maybe this should be renamed as a "check"?

Reply:
yes, was named initially for parsing the vectorized function in the parser while later found out the better place to do it is in the jpy scope implementation. renamed